packages/templatizer/src/generated/workspace.ts

```diff
@@ -44,10 +44,10 @@ export default [
   "scripts": {
     "clean": "lerna run clean",
     "build": "lerna run build --stream",
-    "build:dev": "lerna run build:dev --stream; yarn symlink",
+    "build:dev": "lerna run build:dev --stream; pnpm run symlink",
     "lint": "lerna run lint --parallel",
     "symlink": "symlink-workspace --logLevel error",
-    "postinstall": "yarn symlink"
+    "postinstall": "pnpm run symlink"
```

Comment:
we probably do not need to symlink anymore. Assuming we use `workspace:*` syntax, `pnpm` does the symlinking to simulate local modules

Reply:
yep, symlink is automatically run by pnpm when you do `pnpm install` anyways, this is unnecessary. It should be handled with `workspace:*` protocol